webapp/tests/karma/ts/services/feedback.service.spec.ts

```diff
@@ -46,14 +45,14 @@ describe('Feedback service', () => {
 
   afterEach(() => {
     sinon.restore();
-    clock.restore();
+    clock?.restore();
   });
 
   it('should submit feedback when there is an unhandled error', async () => {
+    clock = sinon.useFakeTimers();
```

Comment:
useFakeTimers and fakeAsync don't work together, because they both try to mock the passage of time. If used together, tests just hang because fakeAsync requires the microtask queue to be empty before finishing the test.